src/class-parsely.php

```diff
@@ -129,6 +131,52 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	/**
+	 * Include a link to the statistics page for an article in the wp-admin Posts List.
+	 * If the post object is the "front page," this will include the main dashboard link instead.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/page_row_actions/
+	 * @see https://developer.wordpress.org/reference/hooks/post_row_actions/
+	 * @param [string[]] $actions  The list of actions to which this function will append
+	 * @param [WP_Post]  $post     The individual post object in the list
```

Comment:
```suggestion
	 * @param array<string, string> $actions The existing list of actions.
	 * @param WP_Post               $post    The individual post object the actions apply to.
```

`string[]` is only for indexed array, like
`[ 'foo', 'bar' ]`. 

What we have in actions, is an associative array, with a string key and a string value: 
`[ 'foo' => 'a link', 'parsely_dash' => 'another link' ]`